hbw/app/assets/javascripts/hbw/widget.js.jsx.coffee

```diff
@@ -9,6 +9,12 @@ modulejs.define(
 
       constructor: (@options) ->
         connection = new Connection(path: @options.widgetPath, host: @options.widgetHost)
+        connection.request(
```

Comment:
pls, move to a separate method, also place the call to the end of the constructor